.github/workflows/docker-ci.yml

```
@@ -1,76 +1,171 @@
+    runs-on: ubuntu-latest
+    outputs:
+      pr_number:  ${{ steps.resolve.outputs.pr_number }}
+      sha:        ${{ steps.resolve.outputs.sha }}
+      authorized: ${{ steps.authz.outputs.authorized }}
+      command:    ${{ steps.resolve.outputs.command }}
+    steps:
+      - name: Resolve PR number, SHA, and command
+        id: resolve
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] "github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)